model_extension/2comp/dialysis/template/in.2comp.template

```diff
@@ -0,0 +1,142 @@
+
+## Ascribing interparticle potentials: ##
+pair_style	hybrid/overlay	lj/expand  	${VLPLJcutoff}		coul/debye	${KAPPA} ${VLPEScutoff}	# LJ cut without delta, kappa, ES cut
+pair_coeff	1	1	lj/expand	1	1.			0	${VLPLJcutoff}		# epsilon, sigma, delta V1-V1, cutoff V1-V1
+pair_coeff	1	1	coul/debye	${VLPEScutoff}							# V1-V1 ES_cut
+pair_coeff	2	2	lj/expand	1	1.			0	${VLPLJcutoff}		# epsilon, sigma, delta V2-V2, cutoff V2-V2
+pair_coeff	2	2	coul/debye	${VLPEScutoff}							# V2-V2 ES_cut
+pair_coeff	1	2	lj/expand	1	1.			0	${VLPLJcutoff}		# epsilon, sigma, delta V1-V2, cutoff V1-V2
+pair_coeff	1	2	coul/debye	${VLPEScutoff}							# V1-V2 ES_cut
+pair_coeff	1	3	lj/expand	1	${SIGMAHC1}	${DELTA1}	${VLP1LinkerLJcutoff}	# epsilon, sigma_hc1, delta_V1-D, cutoff LJ V1-D
+pair_coeff	1	3	coul/debye	${VLPLinkerEScutoff}						# V1-D ES_cut
+pair_coeff	2	3	lj/expand	1	${SIGMAHC2}	${DELTA2}	${VLP2LinkerLJcutoff}	# epsilon, sigma_hc2, delta_V2-D, cutoff LJ V2-D
+pair_coeff	2	3	coul/debye	${VLPLinkerEScutoff}						# V2-D ES_cut
+pair_coeff	3	3	lj/expand	1	${LinkerDiameter}	0	${LinkerLJcutoff} 	# epsilon, sigma_hc, delta D-D, cutoff D-D
+pair_coeff	3	3	coul/debye	${LinkerEScutoff}						# D-D ES_cut
+pair_modify	shift	yes											# the additive e_LJ for repulsion-only   
+
+variable	DeltaTIME	equal	0.000025
+timestep	${DeltaTIME}
+variable	DumpFreqVLP	equal	v_RUNCOUNT/1000
+variable	DumpFreqAll	equal	v_RUNCOUNT/100
+variable	ThermoFreq	equal   v_RUNCOUNT/10000
+variable	RestartFreq	equal   v_RUNCOUNT/5
+
+## Define Computes for Output ##
+compute	VLP1PEVec	VLP1	pe/atom
+compute	VLP2PEVec	VLP2	pe/atom
+compute VLP1KEVec 	VLP1	ke/atom
+compute VLP2KEVec 	VLP2	ke/atom
+compute LinkerPEVec	Linker	pe/atom
+compute LinkerKEVec	Linker	ke/atom
+compute	VLP1PEavg	VLP1	reduce	ave c_VLP1PEVec
+compute	VLP2PEavg	VLP2	reduce	ave c_VLP2PEVec
+compute	VLP1KEavg	VLP1	reduce	ave c_VLP1KEVec
+compute	VLP2KEavg	VLP2	reduce	ave c_VLP2KEVec
+compute	LinkerPEavg	Linker	reduce	ave c_LinkerPEVec
+compute	LinkerKEavg	Linker	reduce	ave c_LinkerKEVec
+
+## Defining Output Information ##
+dump	VLPdata		AllVLPS	  custom		${DumpFreqVLP}	dumpVLP.melt	id	type	x	y	z
+dump	ALLdata		all	      custom		${DumpFreqAll}	dumpAll.melt	id	type	x	y	z
+
+## Ensemble Fixes ##
+## NVT (main simulation) ##
+fix		NVTensemble	all	nvt	temp	1.	1.	0.0025	# T_start, T_stop, T_damp=100*timestep
+
+thermo_style	custom	step temp c_VLP1KEavg c_VLP2KEavg c_LinkerKEavg c_VLP1PEavg c_VLP2PEavg c_LinkerPEavg etotal pe press
+thermo_modify norm no
+thermo	${ThermoFreq}
+restart	${RestartFreq}	Restartc${SALTCONC}.*
+run	${RUNCOUNT}
+
+unfix		NVTensemble
+
+undump		VLPdata
+undump		ALLdata
+
+uncompute	VLP1PEVec
+uncompute	VLP2PEVec
+uncompute	VLP1KEVec
+uncompute	VLP2KEVec
+uncompute	LinkerPEVec
+uncompute	LinkerKEVec
+uncompute	VLP1PEavg
+uncompute	VLP2PEavg
+uncompute	VLP1KEavg
+uncompute	VLP2KEavg
+uncompute	LinkerPEavg
+uncompute	LinkerKEavg
```

Comment:
In general, if you made smaller changes to the files in the template folder, the PR should show only some lines changed. But I am getting a request in your PR to add the full in.2comp.template. It may be that in your repository, you are deleting the in.2comp.template entirely and then adding a new one. I suggesting building on top of the existing files and adding the revised files to your PR.